backends/openvino/quantizer/quantizer.py

```diff
@@ -115,67 +135,90 @@ def set_ignored_scope(
+            qps = [quantization_setup.quantization_points[qid] for qid in quantizer_ids]
+            raise nncf.InternalError(
+                "Different quantization configs are set to one unified scale group:"
+                f"{[(qp.insertion_point.__dict__, str(qp.qconfig)) for qp in qps]}"
             )
-            root_qp = quantization_setup.quantization_points[root_quantizer_id]
 
-            if any(
-                root_qp.qconfig != quantization_setup.quantization_points[q_id].qconfig
-                for q_id in quantizer_ids
-            ):
-                qps = [
-                    quantization_setup.quantization_points[q_id]
-                    for q_id in quantizer_ids
-                ]
-                msg = (
-                    "Different quantization configs are set to one unified scale group:"
-                    f"{[(qp.insertion_point.__dict__, str(qp.qconfig)) for qp in qps]}"
-                )
-                raise nncf.InternalError(msg)
+        root_node = nncf_fx.node_utils.get_graph_node_by_name(graph, root_qp.insertion_point.target_node_name)
+        root_edge_or_node = self._get_edge_or_node(root_node, root_qp, nncf_graph)
 
-            root_target_node = nncf_fx.node_utils.get_graph_node_by_name(
-                graph, root_qp.insertion_point.target_node_name
-            )
-            root_edge_or_node = self._get_edge_or_node(
-                root_target_node, root_qp, nncf_graph
-            )
+        for qid in quantizer_ids:
+            if qid == root_id:
+                continue
 
-            for quantizer_id in quantizer_ids:
-                if quantizer_id == root_quantizer_id:
-                    continue
+            qspec = SharedQuantizationSpec(root_edge_or_node)
+            qp = quantization_setup.quantization_points[qid]
+            edge_or_node, annotation = self._get_edge_or_node_and_annotation(graph, nncf_graph, qp, node_vs_torch_annotation)
+            self._fill_torch_ao_annotation(edge_or_node, qspec, annotation)
 
-                qspec = SharedQuantizationSpec(root_edge_or_node)
-                qp = quantization_setup.quantization_points[quantizer_id]
-                edge_or_node, annotation = self._get_edge_or_node_and_annotation(
-                    graph, nncf_graph, qp, node_vs_torch_annotation
-                )
-                self._fill_torch_ao_annotation(edge_or_node, qspec, annotation)
+    def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+        nncf_graph = nncf_fx.nncf_graph_builder.GraphConverter.create_nncf_graph(model)
+        graph = model.graph
+        node_vs_torch_annotation: DefaultDict[torch.fx.Node, QuantizationAnnotation] = defaultdict(QuantizationAnnotation)
+
+        if self.mode in self.wc_modes:
+            self._annotate_weight_compression(model, graph, nncf_graph, node_vs_torch_annotation)
```

Comment:
Can we return the `node_vs_torch_annotation` from the methods instead of passing it by the reference?